### PR TITLE
IBX-3814: Added checks if the field to upload is type ezimage to add alternative text

### PR DESCRIPTION
**Pull request by mateuszdebinski**
| Question      | Answer
| ------------- | ---
| Tickets       | [IBX-3814](https://issues.ibexa.co/browse/IBX-3814)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Related PR: https://github.com/ezsystems/ezplatform-rest/pull/99

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/ui-dev/src/modules/multi-file-upload/services/multi.file.upload.service.js [hunk 1]
```diff
@@ -147,29 +171,40 @@ const prepareStruct = ({ parentInfo, config, languageCode }, data) => {
...
-                },
-            };
-
-            return struct;
+            const contentType = response.ContentTypeInfoList.ContentType[0];
+            const contentFieldIdentifier = mapping.contentFieldIdentifier;
+
+            return getFieldDefinitionByIdentifier(config, contentType.id, contentFieldIdentifier)
+                .then((response) => response.json())
+                .catch(() => window.eZ.helpers.notification.showErrorNotification('Cannot get content type by identifier'))
```
Comment (GrabowskiM) [inline src/bundle/ui-dev/src/modules/multi-file-upload/services/multi.file.upload.service.js]: Translation

File: src/bundle/ui-dev/src/modules/multi-file-upload/services/multi.file.upload.service.js [hunk 2]
```diff
@@ -147,29 +171,40 @@ const prepareStruct = ({ parentInfo, config, languageCode }, data) => {
...
+
+                    const fields = [
+                        { fieldDefinitionIdentifier: mapping.nameFieldIdentifier, fieldValue: data.file.name },
+                        { fieldDefinitionIdentifier: contentFieldIdentifier, fieldValue: fileValue },
+                    ];
+
+                    const struct = {
+                        ContentCreate: {
+                            ContentType: { _href: contentType._href },
+                            mainLanguageCode: languageCode || parentInfo.language,
```
Comment (GrabowskiM) [inline src/bundle/ui-dev/src/modules/multi-file-upload/services/multi.file.upload.service.js]:
```suggestion
                            mainLanguageCode: languageCode ?? parentInfo.language,
```

File: src/bundle/ui-dev/src/modules/multi-file-upload/services/multi.file.upload.service.js [hunk 3]
```diff
@@ -147,29 +171,40 @@ const prepareStruct = ({ parentInfo, config, languageCode }, data) => {
...
+                            alwaysAvailable: true,
+                            remoteId: null,
+                            modificationDate: new Date().toISOString(),
+                            fields: { field: fields },
+                        },
+                    };
+
+                    return struct;
+                })
+                .catch(() => window.eZ.helpers.notification.showErrorNotification('Cannot create content structure'));
```
Comment (GrabowskiM) [inline src/bundle/ui-dev/src/modules/multi-file-upload/services/multi.file.upload.service.js]: Translation

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

**Review by Gengar-i — APPROVED** [review]

**Review by dew326 — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Review by OstafinL — APPROVED** [review]

**Comment by bogusez** [comment]
Regression tests passed: 
https://github.com/ibexa/experience/pull/118
https://github.com/ibexa/commerce/pull/175
https://github.com/ibexa/content/pull/65

**Review by bogusez — APPROVED** [review]

**Pull request merged**